docs/source/tutorials/multilabel_classification.ipynb

```diff
@@ -0,0 +1,293 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "55b5c376-2c02-43d0-a5fc-ceef6f80b0e2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "issues = find_label_issues(pred_probs=pred_probs,labels=multi_labels,\n",
+    "                           multi_label=True,return_indices_ranked_by='normalized_margin')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c5a41c12-3002-4691-930f-dac5214443bf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def rep_dl(s):\n",
+    "    return os.getcwd()+'/archive/img_align_celeba/img_align_celeba/'+s\n",
+    "\n",
+    "data2 = np.array([rep_dl(i) for i in df.index])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d2b71fa0-ede8-4d16-8f13-da8d98acc90e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_label_name(ls):\n",
+    "    return [class_names[i] for i in ls]\n",
+    "\n",
+    "pred_label_names = [get_label_name(i) for i in pred_labels]\n",
+    "label_names = [get_label_name(i) for i in multi_labels]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fffa88f6-84d7-45fe-8214-0e22079a06d1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w = 10\n",
+    "h = 10\n",
+    "fig = plt.figure(figsize=(32, 32))\n",
+    "columns = 1\n",
+    "rows = 20\n",
+    "ct=0\n",
+    "for i in range(1, columns*rows +1):\n",
+    "    pos = issues[ct]\n",
+    "    label_error = label_names[pos]\n",
+    "    \n",
+    "    path = data2[pos]\n",
+    "    img = torchvision.io.read_image(path).swapaxes(0,2).swapaxes(0,1)\n",
+    "    fig.add_subplot(rows, columns, i)\n",
+    "    plt.axis('off')\n",
+    "    plt.imshow(img)\n",
+    "    plt.text(200, 50, data2[pos].split('/')[-1], bbox=dict(fill=False, edgecolor='black', linewidth=1))\n",
+    "    plt.text(200, 100, label_error, bbox=dict(fill=False, edgecolor='red', linewidth=1))\n",
+    "    plt.text(200, 150, pred_label_names[pos], bbox=dict(fill=False, edgecolor='green', linewidth=1))\n",
+    "    ct+=1\n",
+    "plt.savefig(\"errors.pdf\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d65af827-aeda-4b6b-9ae7-b1f0b84700d5",
+   "metadata": {},
+   "source": [
+    "## Converting multilabels to onehot encoding?\n",
+    "In the multilabel setting, cleanlab expects labels to be a list of lists (length N)\n",
+    "```python\n",
+    "from cleanlab.internal.multilabel_utils import int2onehot\n",
+    "N = num_classes\n",
+    "y_one = int2onehot(labels, K=num_classes)\n",
```

Comment:
```suggestion
    "\n",
    "y_binary_matrix = int2onehot(labels, K=num_classes)\n",
```

Reply:
not clear why you had `N = num_classes` here?

Reply:
You should also demonstrate converting label formats in the reverse direction, as many people originally start with their labels in the binary format.

```
labels_list_format = onehot2int(y_binary_matrix)
```

Reply:
Curtis changed int2onthot to always include number of classes, hence I've added here

Reply:
sorry to clarify:  I wasn't asking about that, I was asking why there was `N = num_classes` line. I assume that was accidentally included, as I don't see `N` being used in the later code. If I am misunderstanding, would be helpful if you can explain what the role of `N` is (should generally not use this variable name for number of classes btw, always use `K` instead)

Reply:
ok got it, thanks for clarifying

Reply:
Fixed: https://github.com/cleanlab/cleanlab/pull/517/commits/7aa53405e8abc5f3f0aa2f2a3a96f90f21516db1